jwst/assign_wcs/tests/test_nircam.py

```diff
@@ -190,12 +190,14 @@ def traverse_wfss_trace(pupil):
     assert orderdet == orderin
 
 
+@pytest.mark.xfail
```

Comment:
A useful feature of `xfail` is to give a reason. This will be helpful to indicate why and when the `xfail` can be removed.
```
xfail(reason='Fails due to JP-XXXX')
```

Reply:
Updated in latest commit.